### DISCUSSION TITLE
syncTable, setSyncDate, getSyncDate ... what for?

### DISCUSSION THREAD

**Discussion by GH20223** [capacitor-community/sqlite#332] [Q&A] [answered]
Hi there,

i don't understand what the functions **syncTable**, **setSyncDate**, **getSyncDate** are supposed to be good for?
What do they do?
When do I have to use them?
Do I need to use them at all?

I want to build an Angular / Capacitor / iOS app. 
Only local database storage.

Can you help me?

Many greetings

**Comment by jepiqueau** [comment, marked as answer]
@GH20223 if you are using local databases you do not need to use them. There are only required if you want to synchronize with a remote server database. Voilà.

**Comment by jepiqueau** [comment]
@GH20223 so i will close the issue happy development for your app.
